evalml/tests/component_tests/test_time_series_regularizer.py

```diff
@@ -152,10 +160,24 @@ def test_ts_regularizer_no_issues(ts_data):
 
 
 @pytest.mark.parametrize("y_passed", [True, False])
-def test_ts_regularizer_X_only(y_passed, combination_of_faulty_datetime):
+def test_ts_regularizer_X_only_equal_payload(y_passed, combination_of_faulty_datetime):
```

Comment:
Just curious what you mean by "equal_payload"

Reply:
This is verifying that if a payload is explicitly passed in through the parameters to the class, it provides an equivalent output to the payload inferred in `fit`.